actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -833,6 +833,27 @@ def fetch(key, *args)
       )
     end
 
+    # Returns parameters for the given keys. If a key can't be found, there are
+    # several options: With no other arguments, it will raise an
+    # ActionController::ParameterMissing error; if a block is given, then that will
+    # be run and its result returned for the missing key.
+    #
+    #     params = ActionController::Parameters.new(name: "Francesco", age: 22)
+    #     params.fetch_values(:name, :age)                # => ["Francesco", 22]
+    #     params.fetch_values(:name, :none)               # => ActionController::ParameterMissing: param is missing or the value is empty or invalid: none
+    #     params.fetch_values(:name, :none) { |key| key } # => ["Francesco", :none]
+    def fetch_values(*keys)
+      values = @parameters.fetch_values(*keys) do |missing_key|
+        missing_key = missing_key.to_sym if missing_key.respond_to?(:to_sym)
+        if block_given?
+          yield missing_key
```

Comment:
The [Hash#fetch_values](https://docs.ruby-lang.org/en/master/Hash.html#method-i-fetch_values) method yields the missing key, so it's good that this implementation does as well.

While reviewing this change, I realized that the `ActionController::Parameters#fetch` method *does not* yield the missing key to the block, despite the fact that [Hash#fetch](https://docs.ruby-lang.org/en/master/Hash.html#method-i-fetch) does.

I've opened https://github.com/rails/rails/pull/56400 to match the built-in behavior.